src/librustdoc/html/static/rustdoc.css

```diff
@@ -475,7 +482,7 @@ h4 > code, h3 > code, .invisible > code {
 }
 #main > .since {
 	top: inherit;
-	font-family: "Fira Sans", sans-serif;
+	font-family: "Fira Sans", Arial;
```

Comment:
Shouldn't we keep sans-serif as Arial may not always be available?

Reply:
That sounds logical indeed. Want to send a PR for it?

Reply:
Oh, it's fixed. https://github.com/rust-lang/rust/commit/7134b0ee6721d25d2e211f01c2f344b0d8e68004

Reply:
Awesome!